Feature/plugins pr

nighcoder opened the pull request:
Pt 6 in [PR #128 comment](https://github.com/clojupyter/clojupyter/pull/128#issuecomment-913001581)

krukow commented on resources/clojupyter/assets/version.edn:
```diff
@@ -1 +1 @@
-{:version "0.3.2"}
+{:version "0.3.3-SNAPSHOT"}
```
Should this match `"0.4.0-SNAPSHOT"` in project.clj?

nighcoder replied:
Yes, we can match it.

As I see it, `version.edn` is read by `version.clj`, which itself was used by the cmdline interface. All those ended up being removed in the fork.

krukow reviewed (approved):
Thanks- Looks good to me!

krukow commented:
@nighcoder - I've approved - I had one question (see above). Let me know when you're ready for this to be merged.

krukow commented:
Thank you!

roryk commented:
Thank you so much both of you. You all rock. I added you as a writer to the repo @nighcoder. Thank you for picking up the torch!

nighcoder commented:
@roryk, thank you!

nighcoder commented:
@krukow, We can merge this.

nighcoder commented:
@krukow I realize I made a mistake by sending this PR to `master` instead of `develop`.

It went against the plan I laid out earlier and I only notice it after I say the commit on the main page.

It's not too bad though, as the previous commits mostly brought extra files and didn't change much in the rest of the project.

Some changes are moving from `deps.edn` to internal managed dependencies by leiningen. This drops the clojure cmdline as a dependency, but will brake anyone who depended on clojupyter's `deps.edn`.

Another change is that I've dropped `lein-v` plugin and the aliases in `project.clj`. I've added those in a later commit.

krukow commented:
I should have caught this too. Thanks for the Ack
